go/pkg/array/map_test.go

```
@@ -0,0 +1,343 @@
+			result := make([]string, len(input))
+			for j, v := range input {
+				result[j] = strconv.Itoa(v)
+			}
+		}
+	})
+
+	b.Run("Append_based", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			var result []string
+			for _, v := range input {
+				result = append(result, strconv.Itoa(v))
```

golangci-lint (2.2.2)

SA4010: this result of append is never used, except maybe in other appends

(staticcheck)